src/js/UserUI.js

```diff
@@ -0,0 +1,24 @@
+import User from './User';
+
+export default class UserUI {
+  constructor() {
+    this.AuthHead = document.querySelector('.auth');
+    this.UserHead = document.querySelector('.user-info');
+    this.userName = document.querySelector('#username-in-head');
+  }
+
+  init() {
+    const { AuthHead, UserHead, userName } = this;
+    const user = User.username;
+    if (User.token) {
+      userName.innerText = user;
+      UserHead.classList.add('active');
+      AuthHead.classList.remove('active');
+      return true;
+    }
+    userName.innerText = '';
+    AuthHead.classList.add('active');
+    UserHead.classList.remove('active');
+    return false;
+  }
```

Comment:
we can refactor this method something like this?
```suggestion
  init() {
    const { AuthHead, UserHead, userName } = this;
    if (User.token) {
      userName.innerText = User.username;
      UserHead.classList.add('active');
      AuthHead.classList.remove('active');
    } else {
      userName.innerText = '';
      AuthHead.classList.add('active');
      UserHead.classList.remove('active');
    }
  }
```

Reply:
yeah i think yes

Reply:
I think Yes, i did it in case if you need some where else get Login status

Reply:
> I think Yes, i did it in case if you need some where else get Login status

Yes, we need login status in some different places and that's why we use emitter